src/sentry/ratelimits/redis.py

```diff
@@ -74,20 +91,23 @@ def current_value(
 
     def is_limited_with_value(
         self, key: str, limit: int, project: Project | None = None, window: int | None = None
-    ) -> tuple[bool, int]:
+    ) -> tuple[bool, int, int]:
         """Does a rate limit check as well as return the new rate limit value"""
+        request_time = time()
```

Comment:
you should update the docstring since you're returning 3 things now and it's not clear what the third thing is until you read through the code